### PR TITLE
Bulk versions and make them stable

### PR DESCRIPTION
**Pull request by kse3hi**
Stabilized all versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,44 +17,44 @@
...
-    "eslint": "^8.54.0",
-    "mocha": "^10.2.0",
-    "mock-fs": "^5.2.0",
-    "nyc": "^15.1.0",
-    "oclif": "^3.17.2",
-    "pkg": "^5.8.1",
-    "prettier": "^3.1.0",
-    "prettier-plugin-organize-imports": "^3.2.4",
-    "shx": "^0.3.4",
-    "sinon": "^15.2.0",
```
[the remark this answers is not in the document]
Comment (kse3hi) [reply package.json:53]: It is already deprecated and as I see doesn't lead for any issues

[the remark this answers is not in the document]
Comment (kse3hi) [reply package.json:53]: `npm WARN deprecated sinon@15.2.0: 16.1.1`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -76,8 +76,6 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v4
-        with:
```
Comment (dennismeister93) [inline .github/workflows/ci.yml:79]: why remove?

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (kse3hi) [reply .github/workflows/ci.yml:79]: It is not clear why we need this.
LInter generates a warning: `Context access might be invalid: ref`
If you know why we need this - let's make a clarification comment.

**Review by dennismeister93 — APPROVED** [review]

**Pull request merged**